docs/design/kotlin-smithy-sdk.md

```diff
@@ -150,60 +141,47 @@ class MyStruct private constructor(builder: BuilderImpl) {
         return "MyStruct(foo=$foo, bar=$bar, baz=$baz, yesno=$yesno)"
     }
 
-    fun toBuilder(): Builder = BuilderImpl(this)
+    fun toBuilder(): Builder = Builder(this)
 
     fun copy(foo: String? = this.foo,
```

Comment:
nit

this isn't actually how we generate `copy`, we must not have updated it. We use the builder implementation.

Reply:
Yeah I started running into things not directly related to the builder change in here and wasn't sure how much to change..